MVVMDemoNew/Catalog/CatalogViewModel.cs

```diff
@@ -53,10 +53,6 @@ private void OnGoToCart()
             {
                 GoToCartRequested.Invoke();
             }
-            else
-            {
-                Trace.WriteLine("Failed to invoke");
-            }
```

Comment:
Always put `Trace.WriteLine()` statement

Reply:
Can you clarify